pyls/plugins/jedi_completion.py

```diff
@@ -1,10 +1,17 @@
 # Copyright 2017 Palantir Technologies, Inc.
 import logging
+
+from distutils.version import LooseVersion
 import parso
+
+import jedi
 from pyls import hookimpl, lsp, _utils
 
+
 log = logging.getLogger(__name__)
 
+JEDI_VERSION = jedi.__version__
```

Comment:
This constant is defined here

https://github.com/palantir/python-language-server/blob/19b10c47988df504872a4fe07c421b0555b3127e/pyls/_utils.py#L13

so please that one instead.

Reply:
See: 7c02479